pkg/cmd/roachprod-microbench/testdata/compare

```diff
@@ -3,34 +3,34 @@ compare name-conflict-a name-conflict-b
 ----
 Package pkg/parent
 Metric B/op
-BenchmarkEntry pkg/parent/sub→WithNameConflict ~ [2200 2000] p=1.000 n=1
```

Comment:
Note: The test data is expected to change as it is ordered by the "name" to be deterministic, which has now changed alphabetic order "new", "old" -> "baseline", "experiment" (where "new" use to be first).